### PR TITLE
feat!:  Load env variables required to build and deploy the app from AWS SSM Parameter Store instead of .env files

### PR DESCRIPTION
**Pull request by pziemkowski**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?

feature

### What is the current behavior?

Currently env stage's variables are loaded from `.env.{stage-name}` files. While this solution works it's quite lacking and unsecure:
- we have a QA env deployed from a mirror of SaaS boilerplate repository so it's difficult to keep it automatically in sync since the `.env.qa` file cannot exist in public repository
- keeping any kind of `.env` files in repo is not a good practice – someone might put a secret variable in repository

### What is the new behavior?

env variables necessary to build and deploy all apps to AWS are now stored in AWS SSM Parameter store.
chamber CLI tool is used to populate system env variables when `nx` commands are run (through custom task runner)
chamber is installed in all CodeBuild projects.

### Does this PR introduce a breaking change?

yes. One needs to move their values stored in `.env.{stage}` to chamber's service named `{stage}`. For example `qa`.

### CODE REVIEW

File: packages/infra/infra-shared/src/stacks/ci/ciBackend.ts [hunk 1]
```diff
@@ -107,6 +108,15 @@ export class BackendCiConfig extends ServiceCiConfig {
       },
       cache: codebuild.Cache.local(codebuild.LocalCacheMode.DOCKER_LAYER),
     });
+
+    project.addToRolePolicy(
```
Comment (mkleszcz) [inline packages/infra/infra-shared/src/stacks/ci/ciBackend.ts:112]: This way backend stack will have access to every KMS keys in the account. WDYT about giving access to the key from bootstrap stack?

File: packages/infra/infra-shared/src/main.ts [hunk 2]
```diff
@@ -11,11 +11,14 @@ import { EnvMainStack } from './stacks/main';
 import { EnvDbStack } from './stacks/db';
 import { EnvComponentsStack } from './stacks/components';
 import { EnvCiStack } from './stacks/ci';
+import { BootstrapStack } from './stacks/bootstrap';
 
 (async () => {
   const envSettings = await loadEnvSettings();
   const app = new App();
 
+  new BootstrapStack(app, 'sb-bootstrap');
```
Comment (pmichalek-apptension) [inline packages/infra/infra-shared/src/main.ts:20]: Maybe let's keep the convention as `new BootstrapStack(app, getGlobalStackName('BoostrapStack', envSettings));`

File: packages/infra/infra-shared/src/stacks/ci/ciBackend.ts (hunk 1 above)
Comment (pziemkowski) [reply packages/infra/infra-shared/src/stacks/ci/ciBackend.ts:112]: Yeah, you are right

File: packages/infra/infra-shared/src/main.ts (hunk 2 above)
Comment (pziemkowski) [reply packages/infra/infra-shared/src/main.ts:20]: We can't really. Think of this stack similarily to the `cdk-bootstrap` stack. There needs to be exactly one instance of it for the whole AWS account. Even if you have multiple projects. That's why I hardcoded the name.

**Review by mkleszcz — APPROVED** [review]

**Pull request merged**